optima/results.py

```diff
@@ -833,21 +874,34 @@ def diff(self, base=None):
                 resultsobj.pops[simkey] -= basepop
         
         return resultsdiff
-    
-    
-    def export(self, filename=None, folder=None, ind=None, writetofile=True, verbose=2, asexcel=True, **kwargs):
-        ''' A method to export each multiresult to a different sheet in Excel (or to a single large text file) '''
+
+    def export(self, filename=None, folder=None, ind=None, writetofile=True, verbose=2, asexcel=True,
+               comparisontab=False, **kwargs):
+        """ A method to export each multiresult to a different sheet in Excel (or to a single large text file) """
         
-        if asexcel: outputdict = odict()
-        else:       outputstr = ''
+        if asexcel:
+            outputdict = odict()
+        else:
+            outputstr = ''
```

Comment:
I assume this was done automatically by PyCharm..I think the original was easier to read though :)

Reply:
Fixed :)